### PR TITLE
docs(Form): update horizontal form example to pf core

### LINKED ISSUE #1577: Update form example
In the [horizontal form example](http://patternfly-react.surge.sh/patternfly-4/components/form) in the workpsace, we should replace the "can we follow up via email" and "remember my password" rows with these recent updates made in the core version - https://pf4.patternfly.org/demos/BasicForms/examples/

@mcarrano do you agree?

### PR DESCRIPTION
**Pull request by boaz0**
**What**:

closes #1577 

//cc @mcoker @mcarrano

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Form/examples/HorizontalForm.js [hunk 1]
```diff
@@ -97,19 +97,13 @@ class HorizontalForm extends React.Component {
             id="horizontal-form-exp"
           />
         </FormGroup>
-        <FormGroup label="Can we follow up via email?" fieldId="horizontal-radio1">
-          <Radio label="Yes" aria-label="yes" id="horizontal-radio1" name="horizontal-radios" />
-          <Radio label="No" aria-label="no" id="horizontal-radio2" name="horizontal-radios" />
+        <FormGroup>
+          <Checkbox label="Follow up via email" id="alt-form-checkbox-1" name="alt-form-checkbox-1" />
+          <Checkbox label="Remember my password for 30 days" id="alt-form-checkbox-2" name="alt-form-checkbox-2" />
```
Comment (mcoker) [inline packages/patternfly-4/react-core/src/components/Form/examples/HorizontalForm.js]: Can you put each of these in their own `FormGroup`?

**Comment by mcarrano** [comment]
Looks good to me.

File: packages/patternfly-4/react-core/src/components/Form/examples/HorizontalForm.js (hunk 1 above)
Comment (boaz0) [reply packages/patternfly-4/react-core/src/components/Form/examples/HorizontalForm.js]: Sure can.

**Comment by boaz0** [comment]
Thank you so much! :+1:

**Pull request merged**